pkglistgen/tool.py

```diff
@@ -575,6 +575,7 @@ def update_and_solve_target(self, api, target_project, target_config, main_repo,
                                 project, scope, force, no_checkout,
                                 only_release_packages, stop_after_solve):
         self.all_architectures = target_config.get('pkglistgen-archs').split(' ')
+        self.use_newest_version = str2bool(target_config.get('pkglistgen-use-newest-version'))
```

Comment:
I fear you're missing a default for all the other projects

Reply:
it simply defaults to false which matches normal behavior

https://github.com/openSUSE/openSUSE-release-tools/blob/8f4795d135733193b7350e7c92c6107149f10c81/osclib/conf.py#L196

Reply:
we can be more explicit, fixed.